iep-nflxenv/src/main/resources/reference.conf

```diff
@@ -75,4 +75,7 @@ netflix.iep.env {
 
   stack = "dev"
   stack = ${?NETFLIX_STACK}
+
+  detail = "unknown"
```

Comment:
This should probably default to an empty string. Detail in most cases is not used and will not be set.

Reply:
ack